### PR TITLE
Refactor operator list

### PR DESCRIPTION
**Pull request by Zishan-7**
- Added sort filter
- Refactored Operator list to match with AVS list implementation.

### CODE REVIEW

File: src/operators/OperatorsList.jsx
Comment (rubo) [inline src/operators/OperatorsList.jsx:1]: Please rename the file to `OperatorList.jsx`

File: src/operators/OperatorsList.jsx [hunk 1]
```diff
@@ -1,198 +1,301 @@
...
-      params.search = debouncedSearchTerm;
-    } else if (page) {
-      params.page = page;
-    } else if (debouncedSearchTerm) {
-      params.page = 1;
-      params.search = debouncedSearchTerm;
-    } else {
-      params.page = 1;
+    params.page = state.searchTriggered ? 1 : page; // If user has searched something update the page number to 1
+    if (debouncedSearchTerm) params.search = debouncedSearchTerm;
```
Comment (rubo) [inline src/operators/OperatorsList.jsx:118]: Let's avoid using single-line `if` statements _everywhere_. They're not readable and, in some cases, seem to cover the line below while it doesn't.

File: src/operators/OperatorsList.jsx [hunk 2]
```diff
@@ -1,198 +1,301 @@
...
-      <div className="mb-1 font-display text-3xl font-medium text-foreground-1">
+    <div className="flex h-full flex-col">
+      <div className="font-display text-3xl font-medium text-foreground-1">
         Operators
       </div>
-      <div className="mb-6 flex w-full flex-col items-center justify-between gap-4 lg:flex-row">
-        <span className="text-foreground-2">
+
+      <div className="mb-4 mt-3 flex w-full flex-col items-start justify-between gap-4 lg:flex-row lg:items-center">
+        <div className="text-foreground-1 lg:w-2/3">
```
Comment (rubo) [inline src/operators/OperatorsList.jsx:150]: The font size is too big for this text. We need `text-xs` here.

File: src/operators/OperatorsList.jsx [hunk 3]
```diff
@@ -1,198 +1,301 @@
...
+            inputWrapper:
+              'border-outline data-[hover=true]:border-foreground-1',
+            input: 'placeholder:text-subtitle'
+          }}
+          value={state.searchTerm ?? ''}
           onChange={handleSearch}
           type="text"
-          placeholder="Search by operator"
+          color="primary"
+          placeholder="Search by Operator"
```
Comment (rubo) [inline src/operators/OperatorsList.jsx:166]: No need to make "operator" capitalized. I even thinking about "Search by name/address".

File: src/operators/OperatorsList.jsx [hunk 4]
```diff
@@ -1,198 +1,301 @@
...
-import { useTailwindBreakpoint } from '../shared/useTailwindBreakpoint';
+import ThirdPartyLogo from '../shared/ThirdPartyLogo';
 import useDebouncedSearch from '../shared/hooks/useDebouncedSearch';
+import { useMutativeReducer } from 'use-mutative';
+import { useServices } from '../@services/ServiceContext';
+
+const columns = [
+  {
+    key: 'operator',
+    label: 'Operators',
```
Comment (rubo) [inline src/operators/OperatorsList.jsx:25]: Must either be "Operator" or "Name".

File: src/operators/OperatorsList.jsx [hunk 5]
```diff
@@ -1,198 +1,301 @@
...
+                >
+                  {column.label}
+                </TableColumn>
+              )}
+            </TableHeader>
+            <TableBody
+              emptyContent={
+                <div className="flex flex-col items-center justify-center">
+                  <span className="text-lg text-foreground-2">
+                    No Operators found for &quot;
```
Comment (rubo) [inline src/operators/OperatorsList.jsx:214]:
```suggestion
                    No operator found for &quot;
```

File: src/operators/OperatorsList.jsx [hunk 6]
```diff
@@ -1,198 +1,301 @@
...
+              {state.isFetchingData
+                ? [...Array(10)].map((_, i) => (
+                    <TableRow key={i} className="border-t border-outline">
+                      <TableCell className="w-2/5 py-6 pe-8 ps-4">
+                        <Skeleton className="h-4 rounded-md bg-default" />
+                      </TableCell>
+                      <TableCell className="w-1/5 py-6 pe-8 ps-4">
+                        <Skeleton className="h-4 rounded-md bg-default" />
+                      </TableCell>
+
```
Comment (rubo) [inline src/operators/OperatorsList.jsx:232]: Please remove redundant whitespaces.

File: src/operators/OperatorsList.jsx [hunk 7]
```diff
@@ -1,198 +1,301 @@
...
+    [setSearchParams]
   );
 
   const handleSearch = e => {
     dispatch({ searchTerm: e.target.value });
   };
 
   useEffect(() => {
-    const page = searchParams.get('page');
+    const page = searchParams.get('page') ?? 1;
```
Comment (rubo) [inline src/operators/OperatorsList.jsx:115]: Please group variable declarations and put a space after them, not between them.

**Review by rubo — CHANGES_REQUESTED** [review]
- When no operator is found, it shows that as an error while it should not.
 - Paddings of list items are different from AVS ones.

File: src/operators/OperatorsList.jsx [hunk 8]
```diff
@@ -1,198 +1,301 @@
...
+  },
+
+  {
+    key: 'tvl',
+    label: 'TVL',
+    className: 'text-end w-40 md:w-2/5'
+  }
+];
 
 const OperatorsList = () => {
```
Comment (rubo) [inline src/operators/OperatorsList.jsx:41]:
```suggestion
export default function OperatorList() {
```

File: src/operators/OperatorsList.jsx [hunk 9]
```diff
@@ -1,198 +1,301 @@
...
-        )}
-      </div>
+          )}
+        </div>
+      )}
     </div>
   );
 };
 
 export default OperatorsList;
```
Comment (rubo) [inline src/operators/OperatorsList.jsx:301]:
```suggestion
```

**Review by rubo — CHANGES_REQUESTED** [review]

File: src/operators/OperatorsList.jsx (hunk 3 above)
Comment (Zishan-7) [reply src/operators/OperatorsList.jsx:166]: "Search by name/address" makes more sense. Should I update it?

File: src/operators/OperatorsList.jsx (hunk 4 above)
Comment (Zishan-7) [reply src/operators/OperatorsList.jsx]: I changed it to"Operator" to align with the AVS table

File: src/operators/OperatorsList.jsx (hunk 3 above)
Comment (rubo) [reply src/operators/OperatorsList.jsx]: Yes, please.

File: src/operators/OperatorsList.jsx (hunk 2 above)
Comment (rubo) [reply src/operators/OperatorsList.jsx]: I disagree but let's keep it to compare to LRT which has that text smaller.

**Review by rubo — APPROVED** [review]

**Pull request merged**